Node/src/models/o365ConnectorCard.ts

```diff
@@ -0,0 +1,577 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+//
+// Microsoft Teams: https://dev.office.com/microsoft-teams
+//
+// Bot Builder Microsoft Teams SDK GitHub
+// https://github.com/OfficeDev/BotBuilder-MicrosoftTeams
+//
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+//
+// MIT License:
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// 'Software'), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED ''AS IS'', WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+
+import * as builder from 'botbuilder';
+import * as teams from './';
+import * as sprintf from 'sprintf-js';
+
+export class O365ConnectorCard implements builder.IIsAttachment {
+    protected data = {
+        contentType: 'application/vnd.microsoft.teams.card.o365connector',
+        content: <teams.IO365ConnectorCard> {}
+    };
+    
+    constructor(protected session?: builder.Session) {
+    }
+
+    public title(text: string|string[], ...args: any[]): this {
+        if (text) {
```

Comment:
Yeah, I just ran into that in Trello. I have a method that creates a card with a tap action, and I needed the same card but with no tap. I figured I could just take the result of the old one and remove the tap action, but nope, had to do more refactoring. Do you know why they chose to add this limitation? I found it surprising.